zuul.d/project.yaml

```diff
@@ -0,0 +1,5 @@
+- project:
+    check:
+      jobs:
+        - staffeln-linters
+        - staffeln-unit
```

Comment:
Is it me or I don't think this job is defined anywhere?

Reply:
it's defined in zuul.d/jobs.yaml https://github.com/vexxhost/staffeln/pull/129/files/aca56c701f2d2f2e4ee150f0979116f4ece811e3#diff-f6856e72c485122d1cb0ed2470e60db446808bd69ccb662517be5c46e4048c4bR6